### ISSUE TITLE
Even More Model2 Disassembly Instruction Display Issues

### ISSUE THREAD

**Issue by cglmrfreeman** [mamedev/mame#6761]
At `0x12720` in the `maincpu` of Sonic the Fighters (`sfight`) the raw data is as follows: `96 4F 29 78`

In IDA Pro, the instruction disassembles (I believe incorrectly, due to no floating point processor being implemented) to the following: `addr    , r5, r5` 

n MAME, the raw opcode is displayed as `78294F96` with the following as the dasm instruction: `78 78:f 78294f96 0 3`

Using onlinedisassembler.com with the i960mc processor set gives us the following disassembly: `addr	0f1.0,r5,r5`

It looks like floating point registers were added to the disassembler, so I do not understand what is wrong:
>https://github.com/mamedev/mame/blob/3ba42de527543564b1f91a79e2cc9754d0a55890/src/devices/cpu/i960/i960dis.cpp#L183

**Comment by ajrhacker** [comment]
The current master version outputs `addr +1.0,r5,r5` for this instruction. Admittedly the sweeping rewrite of the i960 disassembler committed in 704e81105dda7c75c85551b641b2a6ede8facb24 has yet to make it into an official MAME release, but that will happen in time.

**Comment by cglmrfreeman** [comment]
@ajrhacker I was aware of commit 704e81105dda7c75c85551b641b2a6ede8facb24 and really thought I had built from git after that commit (hence the "Even more" in the title), but I am building the latest master to verify, as I have found several other instructions MAME previously did not handle...

Ah, so there it is. You are correct, I shouldn't have opened this. Now let me double check the others.

Yup, they all look good. My mistake, oops!
